docs/releases/2.2.1.txt

```diff
@@ -33,3 +33,5 @@ Bugfixes
 * Reverted an optimization in Django 2.2 (:ticket:`29725`) that caused the
   inconsistent behavior of ``count()`` and ``exists()`` on a reverse
   many-to-many relationship with a custom manager (:ticket:`30325`).
+* On PostgreSQL, :djadmin:`dbshell` now supports client-side TLS certificates
+  and key (:ticket:`30370`).
```

Comment:
This is a new feature, hence it will not be included into `2.2.1` release. Please move release notes to the `3.0`.

Reply:
Moved to the release `3.0`, section `Management Commands`. Is this ok?

Reply:
Sounds good, thanks.